portal-api/src/v1/users/routes.js

```diff
@@ -214,17 +201,18 @@ module.exports.login = async (req, res, next) => {
   const { tokenObject, user } = loginResult;
 
   return res.status(200).json({
-    success: true, token: tokenObject.token, user: sanitizeUser(user), expiresIn: tokenObject.expires,
+    success: true,
+    token: tokenObject.token,
+    user: sanitizeUser(user),
+    expiresIn: tokenObject.expires,
   });
 };
 
 module.exports.resetPassword = async (req, res) => {
   const { email } = req.body;
-  const { success } = await resetPassword(email);
+  await resetPassword(email);
 
-  return res.status(200).json({
-    success,
-  });
+  return res.status(200).send();
```

Comment:
Did we decide that sending `{success: true}` as the response body was pointless here?

Reply:
Yeah -- not much point at the minute. If it's needed, it can be introduced again

Reply:
Sounds good 👍